src/vs/workbench/browser/layout.ts

```diff
@@ -18,7 +18,10 @@ import {IContextViewService} from 'vs/platform/contextview/browser/contextView';
 import {IEventService} from 'vs/platform/event/common/event';
 
 const DEFAULT_MIN_PART_WIDTH = 170;
+const DEFAULT_MIN_PANEL_PART_HEIGHT = 170;
 const HIDE_SIDEBAR_WIDTH_THRESHOLD = 50;
+const HIDE_PANEL_HEIGHT_THRESHOLD = 50;
```

Comment:
@isidorn not sure it would be harder, as a user you can drag from above the status bar to restore it. I think it would be consistent with how the sidebar behaves